tests/printer/reactJsx/__snapshots__/render.spec.js.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I wonder if we should make a separate folder for ppx tests: `tests/ppx/...`? What do you think?